Site/ASAB Maestro/Descriptors/elasticsearch.yaml

```diff
@@ -38,3 +38,56 @@ descriptor:
     xpack.security.transport.ssl.certificate_authorities: certs/ca.pem
     xpack.security.transport.ssl.certificate: certs/certificate.pem
     xpack.security.transport.ssl.key: certs/key.pem
+
+
+nginx:
+  https:
+    location /api/elasticsearch/:
+      - auth_request /_elasticsearch_introspect
+      - auth_request_set $batman $upstream_http_authorization
+      - proxy_set_header Authorization $batman
+
+      - auth_request_set $cookie $upstream_http_cookie
+      - proxy_set_header Cookie $cookie
+
+      - rewrite ^/api/elasticsearch/(.*) /$1 break
+      - proxy_set_header Host $host
+      - proxy_set_header Connection "Keep-Alive"
+      - proxy_set_header Proxy-Connection "Keep-Alive"
+      - proxy_pass http://elasticsearch  # upstream created by remote control technology - it leads to master
+
+    location /_elasticsearch_introspect:
+    - internal
+    - proxy_method POST
+    - proxy_set_body ""
+    - proxy_set_header X-Request-URI "request_uri"
+    - proxy_cache kibana
+    - proxy_pass http://upstream-seacat-auth-private/nginx/introspect/batman?client_id=elasticsearch
+    - proxy_cache_key $cookie_SeaCatSCI_VYNYZBRVIKLGTSQZ  # This is a cookie paired to the elasticsearch client - it contains hashed client_id - this is based on SeaCat Auth requirements.
+    - proxy_cache_lock on
+    - proxy_cache_valid 200 10s
+    - proxy_ignore_headers Cache-Control Expires Set-Cookie
+
+seacat-auth:
+  content:
+  - "cl.json": |
+      [{
+        "_id": "elasticsearch",
+        "application_type": "api",
```

Comment:
@byewokko I didn't know what to fill in, but it worked

Reply:
it's fine like this for now. many of the fields are there just for the sake of the oidc specs, without proper functionality yet.